packages/andromeda-data-storage/src/primitive.rs

```diff
@@ -163,6 +165,11 @@ pub struct GetValueResponse {
     pub value: Primitive,
 }
 
+#[cw_serde]
+pub struct GetTypeResponse {
+    pub value_type: Primitive,
```

Comment:
May have been shortsighted on my end.

I think we should return a `String` here rather than the value and implement `From<String>` for `Primitive`.